src/main/java/org/broadinstitute/hellbender/tools/walkers/readorientation/LearnReadOrientationModel.java

```diff
@@ -0,0 +1,239 @@
+package org.broadinstitute.hellbender.tools.walkers.readorientation;
+
+import com.google.common.annotations.VisibleForTesting;
+import htsjdk.samtools.metrics.MetricsFile;
+import htsjdk.samtools.util.CloserUtil;
+import htsjdk.samtools.util.Histogram;
+import htsjdk.samtools.util.IOUtil;
+import htsjdk.samtools.util.SequenceUtil;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CoverageAnalysisProgramGroup;
+import org.broadinstitute.hellbender.utils.Nucleotide;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Learn the prior probability of read orientation artifact from the output of {@link CollectF1R2Counts}
+ * Details of the model may be found in docs/mutect/mutect.pdf.
+ */
+@CommandLineProgramProperties(
+        summary = "Collect counts of F1R2 reads at each locus of a sam/bam/cram",
+        oneLineSummary = "Collect counts of F1R2 reads at each locus of a sam/bam/cram",
+        programGroup = CoverageAnalysisProgramGroup.class // TODO: check that this is correct
```

Comment:
Address this `TODO`

Reply:
changed to `ShortVariantDiscoveryProgramGroup`